bin/youtube-download

```diff
@@ -13,6 +13,8 @@ my $verbose   = -t STDOUT ? 1 : 0; # don't be verbose on dump terminal
 my $interval  = 1; # sec
 my $proxy     = undef;
 my $interactive = undef;
+my $user_agent  = undef;
+my $dump_user_agent = undef;
```

Comment:
I'm not sure we need the dump-user-agent option.  That seems like something that's just useful for debugging.